src/tools/tidy/src/triagebot.rs

```diff
@@ -0,0 +1,83 @@
+//! Tidy check to ensure paths mentioned in triagebot.toml exist in the project.
+
+use std::path::Path;
+
+use toml::Value;
+
+pub fn check(path: &Path, bad: &mut bool) {
+    let triagebot_path = path.join("triagebot.toml");
+    if !triagebot_path.exists() {
+        tidy_error!(bad, "triagebot.toml file not found");
+        return;
+    }
+
+    let contents = std::fs::read_to_string(&triagebot_path).unwrap();
+    let config: Value = toml::from_str(&contents).unwrap();
+
+    // Check [mentions."*"] sections, i.e. [mentions."compiler/rustc_const_eval/src/"]
+    if let Some(Value::Table(mentions)) = config.get("mentions") {
```

Comment:
I think you should require that these sections exist instead of having an `if let Some`, these sections should be around and if they are not that's a cause for concern and should emit an error

Reply:
How wrong toml would look in the case of `mentions` empty? In general, `mentions` is optional field?

Reply:
The [triagebot docs](https://forge.rust-lang.org/triagebot/mentions.html) don't seem to imply that a `[mentions]` section is mandatory. The repository would just not emit pings to anyone, right? Speaking in general I think it's a valid use-case (maybe we just don't use it).